keel-web/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -30,9 +37,15 @@ import org.springframework.stereotype.Component
  */
 @Component
 class ApplicationService(
-  private val repository: KeelRepository
+  private val repository: KeelRepository,
+  private val constraints: List<ConstraintEvaluator<*>>
 ) {
   private val log by lazy { LoggerFactory.getLogger(javaClass) }
+  private val explicitConstraints: List<ConstraintEvaluator<*>> = constraints.filter { !it.isImplicit() }
```

Comment:
For naming consistency:
```suggestion
  private val explicitConstraintEvaluators: List<ConstraintEvaluator<*>> = constraints.filter { !it.isImplicit() }
```